### DISCUSSION TITLE
Is inter-account transactions supported?

### DISCUSSION THREAD

**Discussion by harshilsharma63** [ananthakumaran/paisa#94] [Q&A]
Lets say I have two savings account, both of which are imported in Paisa. If I transfer funds from my one account to my other account, it shouldn't include that as expense or income. Is this supported in Paisa? Same goes for refunds.

**Comment by ananthakumaran** [comment]
Checking to checking account, for example, `Assets:Checking:ICICI` to `Assets:Checking:AXIS` is already supported. For refund check this discussion https://github.com/ananthakumaran/paisa/discussions/92

**Reply by harshilsharma63** [reply]
Thanks! Will check!

**Reply by harshilsharma63** [reply]
Hmm, it doesn't seem to be working. i have two ledger files - one for each of my HDFC bank accounts. I use `include` to include them both into a single `main.ledger` file. Many inter-bank transfers are showing up as `Expense:Unknown`. I'm sharing the ledger transaction here, anonymized-

Account 1 transaction-
```
2023/03/15 IB FUNDS TRANSFER DR-12345678123456  -HARSHIL SHARMA
    Expenses:Unknown                          1000 INR
    Assets:Checking:HDFC
```

Account 2 transaction-
```
2023/03/15 IB FUNDS TRANSFER CR-12345678123999  -HARSHIL SHARMA
    Assets:Checking:HDFC
    Income:Unknown                           -1000 INR
```

p.s. - the app is super amazing!

**Reply by ananthakumaran** [reply]
I misunderstood your question, I was thinking about the net investment, etc. In the import context, it is a literal transformation (except the predicAccount function). In this case, you would have to delete one of these transactions. PredictAccount could pick up the pattern if you have a distinct description for your inter-account transfer (you would also have to remove the prefix). My suggestion would be not to worry too much about these corner cases when you begin with paisa and just edit the file after import.

**Reply by harshilsharma63** [reply]
but if I remove one transaction, won't it think I made payment to an account? Because its all within my account, I basically want the app to ignore those transactions. Maybe I should delete both?

**Reply by ananthakumaran** [reply]
Depends on the granularity, if you look at the whole assets or assets:checking, it might be ok. But if you look at an individual account, then the balance might not match with bank account balance. Leaving one transaction (two would create duplicates) will make sure the real life bank balance and your account balance matches at all levels.

**Reply by harshilsharma63** [reply]
I'll provide context on why I'm worrying about that it shows up in the yearly cashflow diagram, and also in monthly expenses. I'm trying to use Paisa to understand exactly how much, and from where the money comes in, and more importantly, how much, and where does money go. Since this inter-bank transfer is neither an income, not an expense, ideally I'd want to not show up in cashflow and expense sections. I hope I'm able to explain the issue well.

**Reply by ananthakumaran** [reply]
It's expected to show up in cashflow diagram, but monthly expenses is not expected (no expense accounts are related). Can you double check and share a minimal journal if that's the case. To clarify, you should delete one side and leave the other side with the following transaction, assuming the fund is transferred from hdfc to sbi. It will show up in cashflow diagram, but not in expenses. 

```
2023/03/15 IB FUNDS TRANSFER DR-12345678123456  -HARSHIL SHARMA
    Assets:Checking:SBI                          1000 INR
    Assets:Checking:HDFC
```
